.eslintrc.js

```diff
@@ -7,12 +7,29 @@ module.exports = {
     {
       files: ['*.ts'],
       extends: ['@metamask/eslint-config-typescript'],
+      rules: {
+        // TODO: Move this to our shared config
+        'no-invalid-this': 'off',
+        '@typescript-eslint/no-invalid-this': ['error'],
+      },
     },
 
     {
       files: ['*.js'],
       extends: ['@metamask/eslint-config-nodejs'],
     },
+
+    {
+      files: ['*.test.ts', '*.test.js', './tests/setupAfterEnv.ts'],
```

Comment:
Nit: there are no JS test modules
 ```suggestion
      files: ['*.test.ts', './tests/setupAfterEnv.ts'],
```

Reply:
Ah, forgot this earlier. Thanks!